extension.meta.xml

```diff
@@ -14,6 +14,9 @@
 		</author>
 	</authors>
 	<releases>
+		<release version="1.7.2" date="2017-05-19" min="2.5.x" max="4.x.x" php-min="5.6.x" php-max="7.x.x">
```

Comment:
Strange, I thought that `php-min` and `php-max` was a new "de-facto standard". I vaguely remember that we had a discussion about it, but unfortunately I don't remember where and when.